docs/reference/enable-gpu.rst

```diff
@@ -1,46 +1,153 @@
-Enable Nvidia GPUs
+.. _nvidia-gpu:
+
+Enable NVIDIA GPUs
 ==================
 
-To be able to run workloads on GPU with DSS, you need to have:
+The following section describes how to configure the DSS to utilize the host's
```

Comment:
```suggestion
This guide describes how to configure DSS to utilize the host's
```

Reply:
I would drop the "the" from "the DSS", when using it as an acronym. This is probably something to check with product/marketing. Can't find the way to tag Andreea here. Maybe she doesn't have Github??

Reply:
I am not happy with "host". Does the user know he/she is the host? I would go for:
"This guide describes how to configure DSS to utilize your NVIDIA GPUs."

Reply:
No hard preference on "the", ACK on replacing the word host

Reply:
On a "technical" note. @afgambin the suggestion above will look weird since it puts a period at the end of the sentence, while the next line will continue the sentence. 

If you can make multi-line suggestions then let's do this so that I don't have to make further changes after merging the changes. If not let me know and I'll convert to one liners instead

Reply:
pushed a commit for this d2a2a8a